scripts/model.config.jinja

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+{% set sdf_version = '1.5' if sdf_version == 'NotSet' else sdf_version %}
+<model>
+  <name>{{ model_name }}</name>
+  <version>1.0</version>
+  <sdf version='{{ sdf_version }}'>{{ model_name }}.sdf</sdf>
+
+  <author>
+   <name>Benjamin Perseghetti</name>
+   <email>bperseghetti@rudislabs.com</email>
```

Comment:
We have some models that are under the Apache license. Wouldn't this be problematic if we relabel the model config?